core/src/test/kotlin/org/evomaster/core/output/TestCaseWriterTest.kt

```diff
@@ -0,0 +1,289 @@
+package org.evomaster.core.output
+
+import org.evomaster.core.database.DbAction
+import org.evomaster.core.database.schema.Column
+import org.evomaster.core.database.schema.ForeignKey
+import org.evomaster.core.database.schema.Table
+import org.evomaster.core.problem.rest.RestAction
+import org.evomaster.core.problem.rest.RestIndividual
+import org.evomaster.core.problem.rest.SampleType
+import org.evomaster.core.problem.rest.service.RestFitness
+import org.evomaster.core.search.ActionResult
+import org.evomaster.core.search.EvaluatedIndividual
+import org.evomaster.core.search.FitnessValue
+import org.evomaster.core.search.gene.IntegerGene
+import org.evomaster.core.search.gene.SqlForeignKeyGene
+import org.evomaster.core.search.gene.SqlPrimaryKeyGene
+import org.evomaster.core.search.gene.StringGene
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class TestCaseWriterTest {
+
+    @Test
+    fun testEmptyDbInitialization() {
+
+        val dbInitialization = ArrayList<DbAction>()
+
+        val (format, baseUrlOfSut, ei) = buildEvaluatedIndividual(dbInitialization)
+
+        val test = TestCase(test = ei, name = "test")
+
+        val writer = TestCaseWriter()
+
+        val lines = writer.convertToCompilableTestCode(format, test, baseUrlOfSut)
+
+        val expectedLines = Lines()
+        expectedLines.add("@Test")
+        expectedLines.add("public void test() throws Exception {")
+        expectedLines.add("}")
+
+        assertEquals(expectedLines.toString(), lines.toString())
+    }
+
+    @Test
+    fun testOneAction() {
+        val aColumn = Column("aColumn", "VARCHAR", 10)
+
+        val aTable = Table("myTable", setOf(aColumn), HashSet<ForeignKey>())
+
+
+        val id = 0L
+
+        val gene = StringGene(aColumn.name, "stringValue", 0, 10)
+
+        val insertIntoTableAction = DbAction(aTable, setOf(aColumn), id, mutableListOf(gene))
+
+        val dbInitialization = ArrayList<DbAction>()
+        dbInitialization.add(insertIntoTableAction)
+
+        val (format, baseUrlOfSut, ei) = buildEvaluatedIndividual(dbInitialization)
+
+        val test = TestCase(test = ei, name = "test")
+
+        val writer = TestCaseWriter()
+
+        val lines = writer.convertToCompilableTestCode(format, test, baseUrlOfSut)
+
+        val expectedLines = Lines()
```

Comment:
this can be simplified with ".apply{}", so you do not have to repeat expectedLines